styles.css

```diff
@@ -1,192 +1,782 @@
-*{
+* {
   margin: 0;
   padding: 0;
 }
 
-body{
-  background-color: #D3D3D3;
+body {
+  width: 100%;
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #d3d3d3;
 }
 
-header{
-  display: block;
-  text-align: left;
+.flex {
+  display: flex;
 }
 
-#first-header{
-  display: none;
+#section1 {
+  background-color: #fff;
+  width: 373px;
 }
 
-#logo-header{
+#first-header {
   display: none;
 }
 
+header {
+  display: flex;
+  justify-content: left;
+  width: 375px;
+  position: absolute;
+  background-color: transparent;
+}
 
-#second-header{
+#logo-header {
   display: none;
+  padding: 6px 122px 6px 0;
 }
 
-#section1 img {
+#second-header {
+  display: none;
+}
+
+#hamburg {
   padding-left: 25px;
   padding-top: 20px;
   width: 31px;
   height: 27.28px;
+  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
+  cursor: pointer;
 }
 
-#main-banner{
-  background-image: url("./images/mobile-home-bg.png");
-  background-size: 100% 100%;
+#main-banner {
+  background-image: url("./images/concert-image2.jpg");
   background-position: center;
+  background-blend-mode: darken;
+  background-color: rgba(0, 0, 0, 0.5);
+  height: 486px;
 }
 
-#logo-header{
-  padding: 6px 122px 6px 0px;
-  
+#navbar-h2 {
+  display: none;
 }
 
-#navbar-h2{
- display: none;
+#main-content h4 {
+  padding: 75px 0 0 16px;
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+  font-weight: 400;
 }
 
-#main-banner{
-  height: 486px;
+#text-image {
+  padding: 0 0 16px 16px;
+  font-family: 'Lato', sans-serif;
+  font-weight: 900;
+  font-size: 30px;
+
+  /* color: transparent; */
+  color: #ec5242;
+  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
+
+  /* background-image: url(./images/orangebg.jpg); */

```

Comment:
Great design @rodrigopenayo98 🤝

- [ ] [Optional] Kindly remove all commented code from your project as this is regarded as a bad practice.